src/etc/inc/config.console.inc

```diff
@@ -532,8 +532,7 @@ function vlan_setup() {
 		$vlan['if'] = chop(fgets($fp));
 
 		if ($vlan['if']) {
-			if (!array_key_exists($vlan['if'], $iflist) or
-			    !is_jumbo_capable($vlan['if'])) {
+			if (!array_key_exists($vlan['if'], $iflist) || !is_jumbo_capable($vlan['if'])) {
```

Comment:
This is, quite possibly, one of my pet peeves: code that overflows 80 columns and so I can't hack it well on a netbook-type device. Also what rbgarga said about code quality. If you have too many nested conditionals, you're doing it wrong, IMO.